app/controllers/application_controller.rb

```diff
@@ -2,7 +2,7 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
   def current_user
-    @current_user ||= User.find(session[:user_id])
+    @current_user ||= User.find_by(id: session[:user_id])
```

Comment:
if this gets set to `nil`, it'll attempt to do a look up every time. might be something to keep an eye on as the usage of `current_user` grows.